processors/user-group.ftd

```diff
@@ -0,0 +1,31 @@
+-- ft.page: `user-groups` processor
```

Comment:
Every file you add must have an entry in sitemap also else there would be no link to it. I think the latest processor your write, full-sitemap also has no entry in sitemap so I had to ask you for the URL.